### DISCUSSION TITLE
Is this project down or abandoned ?

### DISCUSSION THREAD

**Discussion by sidahmed-malaoui** [dstotijn/hetty#59] [General] [answered]
@dstotijn  This project was really a hype for me in the beginning. But now, I see that it's not moving at all.

What's happening ? When will we get a version where we can at least intercept requests and edit them ?

Is the project on hold because you didn't find much help from the community ? If it's the case, how can I/we help ?

Thank you in advance for your response.

**Comment by dee-see** [comment]
I'm not a contributor to the project, but there was a release 2 days ago and it's the Christmas holidays right now. I don't think the project is abandoned. ;)

**Comment by dstotijn** [comment, marked as answer]
> @dstotijn This project was really a hype for me in the beginning. But now, I see that it's not moving at all.
> 
> What's happening ? When will we get a version where we can at least intercept requests and edit them ?
> 
> Is the project on hold because you didn't find much help from the community ? If it's the case, how can I/we help ?
> 
> Thank you in advance for your response.

Hi, had to drop the pace on the project last few months because of other commitments but it's definitely not abandoned. Wanted to scaffold the search feature first, which is now done finally in v0.3.0. Next up I'm going to prep/poc the req/res intercept module. Check back in a few weeks, or feel free to hang out in the Discord server; I'll likely ask for review/discuss things on the intercept feature in there.

**Reply by sidahmed-malaoui** [reply]
I see dude. 

I hope I wasn't harsh on you, because even if you abondon the project, we would still be thankful for your attempt and for your help to the community.

It's just that my expectations are really high for this project, and I'm really looking forward to use it and contribute in it.

For discord, I would definitely love to join the team. Can you give me your discord channel please ?

**Reply by dstotijn** [reply]
> I see dude.
> 
> I hope I wasn't harsh on you, because even if you abondon the project, we would still be thankful for your attempt and for your help to the community.
> 
> It's just that my expectations are really high for this project, and I'm really looking forward to use it and contribute in it.
> 
> For discord, I would definitely love to join the team. Can you give me your discord channel please ?

No worries, it's all good! The link for Discord: https://discord.gg/3HVsj5pTFP
